code/build.gradle.kts

```diff
@@ -0,0 +1,12 @@
+buildscript {
```

Comment:
Are you forgetting the `aep-license` here?
```
apply(plugin = "aep-license")
```

Reply:
I have this in the next PR.